### ISSUE TITLE
Improve startup message

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/briefcase#358]
I think the ODK start up message could be friendlier! I've drafted a first pass...
```
Welcome to ODK Briefcase! Here are three things you should know to get started.

   1. You must set a Storage Location where Briefcase will store data that it needs to operate. You will not be able to use Briefcase until you set this location.
   2. We gather anonymous usage data (e.g., operating system, version number) to help our community prioritize fixes and features. If you do not want to contribute your data, please disable that setting.
   3. ODK is a community-powered project and the community lives at https://forum.opendatakit.org. Stop by for a visit and introduce yourself!
```

**Comment by darshan934** [comment]
This is an easy fix I am working on it. Are there any specific things that I must mention as part of the welcome message?

**Comment by yanokwa** [comment]
Fixed by #415
